src/modules/hunting/cves.py

```diff
@@ -61,6 +62,16 @@ def __init__(self, binary_version):
         self.evidence = "kubectl version: {}".format(self.binary_version)
 
 
+""" kube-state-metrics CVE """
+
+class KubeStateMetricsVulnerability(Vulnerability, Event):
+    """"""
+    def __init__(self, version):
+        Vulnerability.__init__(self, KubectlClient, "kube-state-metrics Vulnerable To CVE-2019-17110", category=InformationDisclosure, vid="KVH051")
+        self.version = version
+        self.evidence = "kubectl version: {}".format(self.version)
```

Comment:
if you decide the vulnerable component is kubestatemetrics (see my other comment), then this should change accordingly

Reply:
Yes, correct. same mistake, apologies.